test/attributes.test.ts

```diff
@@ -166,8 +165,187 @@ describe("Execute AttributeQueries", () => {
                 }
             }
         });
+
         expect(executeRelationshipAttributeQueryResult).toBeSuccessful(ValidationSchema.ConnectorAttribute);
 
         expect(executeRelationshipAttributeQueryResult.result.content.value.value).toBe("AString");
     });
+
+    // TODO: Tests lead to timout errors in the syncUntilHasMessages method => remove tests?
```

Comment:
Could it be, that another sync (another worker?) called the /sync route and stole the message?

Reply:
Possible. We haven't found the culprit yet.